articles/openshift/oshift-how-add-domains-proxy-whitelist.md

```diff
@@ -33,11 +33,22 @@ You must also have access to either:
 
 Squid proxy is installed as a service on the control plane load balancers. Within the Squid configuration, we have specified an ACL (Access Control List) as being a list of destination domains (present in a file on the file system), which we will refer to as the whitelist for the remainder of this article. Outbound traffic from the cluster's internal network will pass through the proxy with the destination domain being compared against the whitelist for each connection attempt; if the domain exists then outbound internet traffic is permitted, otherwise it is denied with a HTTP 403 error.
 
+By default, the following domains are added to the whitelist (and cannot be removed) to facilitate installation, testing and ongoing operation of the cluster:
+```
+registry.access.redhat.com
+registry.redhat.io
+.ukcloud.com
+.docker.io
+.docker.com
+.rubygems.org
+.github.com
+```
+
 A scheduled job (which runs at 0 minutes past every hour) on the OpenShift cluster Bastion host reads a Config Map named `proxy-whitelist` within the `whitelist` project. If there are any modifications to this Config Map, the job overwrites the previous custom entries within the whitelist and triggers a reconfigure task on the Squid proxy to enable the updated domains to be accessed.
 
 ### Assigning non cluster-admin users rights to edit the whitelist 
 
-By default, only users who have been assigned the cluster-admin role will be able to view/edit the `proxy-whitelist` ConfigMap within the `whitelist` project. To allow non cluster-admin users to edit this object, use the following Role Based Access Control (RBAC) commands:
+By default, only users who have been assigned the cluster-admin role will be able to view/edit the `proxy-whitelist` ConfigMap within the `whitelist` project. Granting a user cluster-admin rights should be done with great consideration as you should always enforce the [principle of least privilege](https://en.wikipedia.org/wiki/Principle_of_least_privilege). To allow non cluster-admin users to edit this object, use the following Role Based Access Control (RBAC) commands which abide by this principle:
```

Comment:
This wasn't quite what I meant - I was meaning that giving a non-admin the rights to alter the whitelist was in effect giving them significant cluster-wide control and should be carefully considered. Rather than talking about granting cluster-admin rights which you've talked about here.  We definitely should talk about principle of least privilege and not handing our rights, but this should be in the security best practice we need to do for the ITHC outcomes.

Reply:
Amended to better reflect danger of giving rights to a non-admin user

Reply:
Simplified language